deployment/aws-fms-automations.template

```diff
@@ -326,27 +327,11 @@
               ]
             },
             "LOG_LEVEL": "info",
-            "CUSTOM_SDK_USER_AGENT": {
-              "Fn::Join": [
-                "",
-                [
-                  "AwsSolution/",
-                  {
-                    "Fn::FindInMap": [
-                      "CommonResourceStackMap",
-                      "Solution",
-                      "SolutionId"
-                    ]
-                  },
-                  "/",
-                  {
-                    "Fn::FindInMap": [
-                      "CommonResourceStackMap",
-                      "Solution",
-                      "SolutionVersion"
-                    ]
-                  }
-                ]
+            "USER_AGENT_PREFIX": {
+              "Fn::FindInMap": [
+                "CommonResourceStackMap",
+                "Solution",
+                "UserAgentPrefix"
```

Comment:
looks like wrong user agent prefix? shouldn't it follow the standard `AwsSolution/<SolutionId>/<SolutionVersion>`